crates/nu-command/src/commands/filters/select.rs

```diff
@@ -154,3 +162,65 @@ fn select(args: CommandArgs) -> Result<OutputStream, ShellError> {
     }))
     .into_output_stream())
 }
+
+#[cfg(test)]
+mod tests {
+    use nu_protocol::ColumnPath;
+    use nu_source::Span;
+    use nu_source::SpannedItem;
+    use nu_source::Tag;
+    use nu_stream::InputStream;
+    use nu_test_support::value::nothing;
+    use nu_test_support::value::row;
+    use nu_test_support::value::string;
+
+    use super::select;
+    use super::Command;
+    use super::ShellError;
+
+    #[test]
+    fn examples_work_as_expected() -> Result<(), ShellError> {
+        use crate::examples::test as test_examples;
+
+        test_examples(Command {})
+    }
```

Comment:
`select` didn't have examples test run before. If that was on purpose I can remove this test.